admin_guide/scheduling/scheduler.adoc

```diff
@@ -634,64 +628,53 @@ concentration of pods.
    ]
 ----
 <1> Specify a name for the priority.
-<2> Specify a weight from 1 (bad fit) to 10 (best fit).
+<2> Specify a weight. Weight can be any non-zero positive value.
```

Comment:
Is there no longer a range for valid weights?
s/Weight can be/Enter

Reply:
@kalexand-rh No. This was incorrect.